opentaskspal/src/main/java/org/dmfs/opentaskspal/rowsets/Subtasks.java

```diff
@@ -36,11 +38,12 @@
 public final class Subtasks extends DelegatingRowSet<Tasks>
 {
 
+    @SuppressWarnings("unchecked")
     public Subtasks(@NonNull View<Tasks> view,
-                    @NonNull Projection<? super Tasks> projection,
+                    @NonNull Projection<Tasks> projection,
                     @NonNull RowReference<Tasks> parentTask)
     {
-        super(new QueryRowSet<>(view, projection, new ReferringTo<>(Tasks.PARENT_ID, parentTask)));
+        super(new QueryRowSetSorting<>(view, projection, new ReferringTo<>(Tasks.PARENT_ID, parentTask), new Present(Tasks.STATUS + ", " + Tasks.TITLE)));
```

Comment:
The custom `QueryRowSetSorting` is not necessary. You can apply the `Sorted` decorator to `view` giving you the same result. Check this out:
```
 super(
    new QueryRowSet<>(
        new Sorted<>(Tasks.STATUS + ", " + Tasks.TITLE, view),
        projection,
        new ReferringTo<>(Tasks.PARENT_ID, parentTask)));
```

Reply:
Uh, that's awesome, thank you, I will adjust these changes tomorrow